apps/vs-code-designer/src/app/commands/createNewProject/createNewProject.ts

```diff
@@ -73,6 +73,7 @@ export async function createNewProjectInternal(context: IActionContext, options:
   await wizard.execute();
 
   await createArtifactsFolder(context as IFunctionWizardContext);
+  await createLibFolder(context as IFunctionWizardContext);
```

Comment:
I wonder if this lib folder needs to be always here whenever the user creates a new project. With the demo It seems that its specific to a NuGet-based logic app. Can this lib folder be created when it is converted to NuGet based?